internal/policies/client.go

```diff
@@ -143,17 +180,56 @@ func (f *Client) listClusterAdmissionPolicies(ctx context.Context) ([]policiesv1
 	return clusterAdmissionPolicyList.Items, nil
 }
 
+// listClusterAdmissionPolicyGroups returns all the ClusterAdmissionPolicyGroups in the cluster.
+func (f *Client) listClusterAdmissionPolicyGroups(ctx context.Context) ([]policiesv1.ClusterAdmissionPolicyGroup, error) {
+	var clusterAdmissionPolicyGroupList policiesv1.ClusterAdmissionPolicyGroupList
+
+	err := f.client.List(ctx, &clusterAdmissionPolicyGroupList)
+	if err != nil {
+		return nil, fmt.Errorf("cannot list ClusterAdmissionPolicies: %w", err)
+	}
+
+	return clusterAdmissionPolicyGroupList.Items, nil
+}
+
 // listAdmissionPolicies returns all the AdmissionPolicies in the given namespace.
-func (f *Client) listAdmissionPolicies(ctx context.Context, namespace string) ([]policiesv1.AdmissionPolicy, error) {
+func (f *Client) listAdmissionPolicies(ctx context.Context, nsName string) ([]policiesv1.AdmissionPolicy, error) {
```

Comment:
All the `findXXX` functions signatures have been changed by this PR to take a Namespace object instead of its name. Should we be consistent here as well?